Terminal.Gui/Core/View.cs

```diff
@@ -124,6 +124,16 @@ internal enum Direction {
 
 		TextFormatter viewText;
 
+		/// <summary>
+		/// Event fired when a subview is being added to this view.
+		/// </summary>
+		public Action<View> Adding;
```

Comment:
per the other PR on this topic:

>On names: The actual implementation is synchronous and the call to the virtual method (or the event) happens as the view is being added. In the PR, the call to the virtual function happens AFTER the call to subviews.Add. Thus the proper tense is "past tense". Thus the name of the virtual function (and event if it's required) should be past tense: "Added".

>Since this refers to another method, where "View" is not included in the name, we should not include "View" in the name here either. Thus the virtual function, being consistent with the rest of the project, should be "OnAdded".

Reply:
@tig be consistent in what you say please. This change @fergusonr merged from my PR #723 and you criticized me for using `Added `and `OnAdded `and are now saying the opposite. That way you discourage anyone who has the slightest desire to make improvements to this project.

Reply:
You are right: I changed my mind between the original code review and subsequent discussions with Miguel that made see I was wrong before. 

In my opinion, adding new APIs should be "highly considered". We should all be open to having strong opinions and then debating things. We should be willing to be wrong. And we should be willing to change our minds if further debate and thought shows we were wrong.

If we were just working on an "App" I'd feel less strongly about getting issues like this right. But we are creating an API and thus the bar is higher. I view it as one of the things that makes working on this project so much fun.

Reply:
> I view it as one of the things that makes working on this project so much fun.

@tig but commenting on a PR that just merged a PR of mine is not that fun. At least you could admit you were wrong in my PR instead of commenting here.

Reply:
> > I view it as one of the things that makes working on this project so much fun.
> 
> @tig but commenting on a PR that just merged a PR of mine is not that fun. At least you could admit you were wrong in my PR instead of commenting here.

I was hoping you and @fergusonr were working together on this PR. I had asked that you do so. 

I made the same points on the other threads that I made here about naming and not adding an API for the removed case.